webpack.config.js

```diff
@@ -308,6 +309,10 @@ const webpackConfig = {
 			),
 		} ),
 		isCalypsoClient && new InlineConstantExportsPlugin( /\/client\/state\/action-types.js$/ ),
+		new CopyPlugin([
+			{ from: path.resolve( __dirname, 'node_modules/@ampproject/worker-dom/dist/worker/worker.mjs' ), to: path.resolve( __dirname, 'public/workerdom/worker.mjs' ) },
+			{ from: path.resolve( __dirname, 'node_modules/@ampproject/worker-dom/dist/worker/worker.mjs.map' ), to: path.resolve( __dirname, 'public/workerdom/worker.mjs.map' ) },
```

Comment:
I was trying this and this step didn't work: I had to create `public/workerdom` and copy the files there manually